pkg/accounting/rollup/rollup.go

```diff
@@ -102,12 +106,12 @@ func (r *Service) RollupStorage(ctx context.Context, lastRollup time.Time, rollu
 	//loop through tallies and build Rollup
 	for _, tallyRow := range tallies {
 		node := tallyRow.NodeID
-		if tallyRow.CreatedAt.After(latestTally) {
-			latestTally = tallyRow.CreatedAt
+		interval := tallyRow.IntervalEndTime.UTC()
+		if interval.After(latestTally) {
+			latestTally = interval
 		}
 		//create or get AccoutingRollup day entry
-		iDay := tallyRow.IntervalEndTime
-		iDay = time.Date(iDay.Year(), iDay.Month(), iDay.Day(), 0, 0, 0, 0, iDay.Location())
+		iDay := time.Date(interval.Year(), interval.Month(), interval.Day(), 0, 0, 0, 0, interval.Location())
```

Comment:
let's add interval.Hour() back in here since we may keep the 6 hr rollup interval